app/Models/classe.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class classe extends Model
```

Comment:
Hello Noha,,,

I think you have suffered from using `Class` as model name, there is a little trick here, if you cannot give the model the same name of migration file you can add to your model 

```php
protected $table = 'table name'
```

this will do the job

Reply:
Thank you soooooooooooooo much for your comments.